docs/pages/toolpad/core/api/dashboard-sidebar-page-item.json

```diff
@@ -0,0 +1,31 @@
+{
+  "props": {
```

Comment:
🤔  I would have expect this to just have a `item` prop. Does it need all these other props? This is just to render the default item right? I'm not sure we should expose more API on it.

Reply:
If you look at the expanded demo [here](https://deploy-preview-4809--mui-toolpad-docs.netlify.app/toolpad/core/react-dashboard-layout/#custom-page-items) I was just trying to make this type of changes as easy as possible as well as reusing the existing component.

Would it help if these props were additional options maybe? And the first parameter was the item?